src/bootstrap_cache.rs

```diff
@@ -30,7 +30,7 @@ const MAX_CACHE_SIZE: usize = 200;
 
 /// A very simple LRU like struct that writes itself to disk every 10 entries added.
 #[derive(Debug, Clone)]
-pub struct BootstrapCache {
+pub(crate) struct BootstrapCache {
```

Comment:
This whole thing can also go as we can use PrefixMap to get old contacts to bootstrap from. Perhaps worth tie in with @grumbach with help from @lionel1704 and we can crush this one too.

Reply:
I'll take this as a follow-up! Don't want to pollute the PR.